code/restful-api/restful-api/Models/InstituicaoFinanceira.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace RestfulApi.Models
+{
+    public class InstituicaoFinanceira
```

Comment:
Is this class related to anything? Could not find it.

Reply:
Retirado. Substituido pelo IPayPal